apps/docs/docs/reference/ui/organisms/OutOfStock.mdx

```diff
@@ -0,0 +1,74 @@
+---
+description: Component that displays a subscription form to alert shoppers when the product is back in stock.
+keywords:
+  - UI library
+  - ecommerce
+  - react
+---
+
+import PropsSection from "@site/src/components/PropsSection/PropsSection";
+
+`OutOfStock` displays a subscription form for shoppers who want to be alerted when the product is back in stock.
+
+The `OutOfStock` component is a compound of the following:
+
+- `OutOfStock`: wraps the out of stock component with title and message;
+- `OutOfStockTitle`: the main title component for out of stock component;
+- `OutOfStockMessage`: the message component for out of stock component;
+
+## Import
+
+```tsx
+import {
+  OutOfStock,
+  OutOfStockTitle,
+  OutOfStockMessage,
+} from '@faststore/ui'
+```
+
+## Usage
+
+### Default
+
+```tsx live
+     <OutOfStock>
+    <OutOfStockTitle>
+      Text <span>icon</span>
+    </OutOfStockTitle>
+    <OutOfStockMessage>Notify me when available</OutOfStockMessage>
+    <Label>
+      Email
+      <Input />
+    </Label>
+    <Button>Notify me</Button>
+  </OutOfStock>
+```
+
+## Props
+
+<PropsSection name="OutOfStock" />
+
+
+## Customization
+`data-store-out-of-stock`
+
+`data-out-of-stock-form`
+
+`data-out-of-stock-title`
+
+`data-out-of-stock-message`
+
+
+## Best practices
+
+### Do's
+- Use the component in a PDP. 
+- Inform the user when their email is incorrect, e.g. *Please enter a valid email address*.
+
+
+### Don'ts
+- Don't use the `OutOfStock` for alert messages, instead use the [`Alert` component](https://www.faststore.dev/reference/ui/molecules/Alert).
```

Comment:
```suggestion
- Don't use the `OutOfStock` for alert messages, instead use the [Alert component](https://www.faststore.dev/reference/ui/molecules/Alert).
```

I think we leave it without backticks to follow the others components doc pattern.